go/bundle/README.md

```diff
@@ -40,7 +40,7 @@ These command-line flags are common to all the three options:
 
 #### From a HAR file
 
-One convenient way to generate HAR file is via Chrome Devtools. Navigate to "Network" panel, and right-click on any resource and select "Save as HAR with content".
+One convenient way to generate HAR file is via Chrome Devtools. Navigate to "Network" panel, and right-click on any resource and select "Save as HAR with content" (before saving, make sure that the "Disable cache" option is checked in the same "Network" panel).
```

Comment:
Checking "Disable cache" after loading resources does not have any effect to HAR generation. So the right procedure is:

1. Open DevTools, navigate to Network panel
2. Make sure "Disable cache" is checked
3. Reload the page
4. "Save as HAR with content"